src-frontend/app/controllers/procedure.js

```diff
@@ -63,6 +64,29 @@ export default Ember.ObjectController.extend({
             });
 
             this.endPropertyChanges();
+        },
+        generateProtocol: function() {
+            var filename = this.get('model.title') + '.xml';
+            $.ajax({
+                beforeSend: function(xhr) {
+                    xhr.setRequestHeader('Authorization', 'Token ' + Cookies.get('authorizationToken'));
```

Comment:
I didn't do that, wasn't in the scope of any tasks I've done. My requests for things like signup when through cause we were hitting `/auth/` which doesn't require a token.